vendor/github.com/opencontainers/runtime-spec/specs-go/config.go

```diff
@@ -178,7 +178,7 @@ type Linux struct {
 	// MountLabel specifies the selinux context for the mounts in the container.
 	MountLabel string `json:"mountLabel,omitempty"`
 	// IntelRdt contains Intel Resource Director Technology (RDT) information for
-	// handling resource constraints (e.g., L3 cache, memory bandwidth) for the container
```

Comment:
You can't and shouldn't patch `vendor/` in place. This change belongs to github.com/opencontainers/runtime-spec repo.

OK, I found that this is the subject of https://github.com/opencontainers/runtime-spec/pull/1076 -- once merged, we need to revisit this PR.